.github/workflows/code-review.yaml

```diff
@@ -35,8 +35,18 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable
 
-      - name: Determine Turbo Cache Path
-        uses: ./.github/workflows/determine-turbo-cache-path.yaml
+      - name: Determine Turbo version
+        id: determine-turbo-version
+        run: |
+          TURBO_VERSION=$(yarn turbo --version)
+          echo "Turbo version: $TURBO_VERSION"
+          if [[ "$TURBO_VERSION" =~ ^1\..* ]]; then
+            TURBO_CACHE_PATH="node_modules/.cache/turbo"
+          else
+            TURBO_CACHE_PATH=".turbo"
+          fi
+          echo "TURBO_CACHE_PATH=$TURBO_CACHE_PATH" >> $GITHUB_ENV
+          echo "Using cache path: $TURBO_CACHE_PATH"
```

Comment:
maybe can we explicitly set `--cache-dir` (or `TURBO_CACHE_DIR`) to maintain the same directory

Reply:
https://turbo.build/repo/docs/reference/run#--cache-dir-path

Reply:
I've tested forcing the cache dir using the env and it works!
Test using turbo 1.x.x and cache dir like in 2.x.x: https://github.com/pagopa/io-ipatente/actions/runs/10844985008/job/30094904145

Reply:
done in dx template